concepticondata/conceptlists/Anonby-2018-1500.tsv

```diff
@@ -0,0 +1,1501 @@
+Anonby-2018-1500-1023	1023	922	PEAR	pear	گلابی	golābi	Plants	Plant products	Fruits
+Anonby-2018-1500-1024	1024	2043	PEACH	peach	هلو	holu	Plants	Plant products	Fruits
+Anonby-2018-1500-1025	1025	3732	POMEGRANATE	pomegranate	انار	anār	Plants	Plant products	Fruits
+Anonby-2018-1500-1026	1026			quince	به	beh	Plants	Plant products	Fruits
+Anonby-2018-1500-1027	1027	904	BERRY	berry (mulberry)	توت	tut	Plants	Plant products	Fruits
+Anonby-2018-1500-1028	1028	322	FIG	fig	انجیر	anjir	Plants	Plant products	Fruits
+Anonby-2018-1500-1029	1029	3801	DATE (FRUIT)	date	خرما	xormā	Plants	Plant products	Fruits
+Anonby-2018-1500-1030	1030	986	TOMATO	tomato	گوجه فرنگی	gaʋjefarangi	Plants	Plant products	Vegetables
+Anonby-2018-1500-1031	1031	2366	ONION	onion	پیاز	piyāz	Plants	Plant products	Vegetables
+Anonby-2018-1500-1032	1032	2368	GARLIC	garlic	سیر	sir	Plants	Plant products	Vegetables
+Anonby-2018-1500-1033	1033			pepper (green)	فلفل سبز، فلفل دلمه‌ای	felfelsabz, felfeldolme’i	Plants	Plant products	Vegetables
+Anonby-2018-1500-1034	1034	919	CHILI PEPPER	red pepper, hot pepper	فلفل قرمز	felfelġermez / felfel-e ġermez	Plants	Plant products	Vegetables
+Anonby-2018-1500-1035	1035	1737	OKRA	okra	بامیه	bāmiye	Plants	Plant products	Vegetables
+Anonby-2018-1500-1036	1036	1146	AUBERGINE	eggplant	بادمجان	bādemjān	Plants	Plant products	Vegetables
+Anonby-2018-1500-1037	1037	641	MUSHROOM	mushroom	قارچ	ġarč	Plants	Plant products	Vegetables
+Anonby-2018-1500-1038	1038	3085	CARROT	carrot	هویج	haʋij	Plants	Plant products	Tubers
+Anonby-2018-1500-1039	1039	593	POTATO	potato	سیب زمینی 	sibzamini	Plants	Plant products	Tubers
+Anonby-2018-1500-1040	1040	506	MAIZE	maize, corn	بلال، ذرت	balāl, zorat	Plants	Plant products	Cereals
+Anonby-2018-1500-1041	1041	1077	WHEAT	wheat	گندم	gandom	Plants	Plant products	Cereals
+Anonby-2018-1500-1042	1042	932	BARLEY	barley	جو	jaʋ	Plants	Plant products	Cereals
+Anonby-2018-1500-1043	1043	926	RICE	rice	برنج	berenj	Plants	Plant products	Cereals
+Anonby-2018-1500-1044	1044	2364	PEANUT	peanut	بادام زمینی	bādāmzamini	Plants	Plant products	Other plant products
+Anonby-2018-1500-1045	1045	2797	SESAME	sesame seed	کنجد	konjed	Plants	Plant products	Other plant products
+Anonby-2018-1500-1046	1046	2042	WALNUT	walnut	گردو	gerdu	Plants	Plant products	Other plant products
+Anonby-2018-1500-1047	1047	968	ACORN	acorn	بلوط	balut	Plants	Plant products	Other plant products
+Anonby-2018-1500-1048	1048			squash seed	تخمِ کدو، تخمه سفید	toxm-e kadu, toxmesefid	Plants	Plant products	Other plant products
+Anonby-2018-1500-1049	1049	1299	COFFEE	coffee	قهوه	ġahʋe	Plants	Plant products	Other plant products
+Anonby-2018-1500-1050	1050	1373	TEA	tea	چای	čāy	Plants	Plant products	Other plant products
+Anonby-2018-1500-1051	1051	1850	COTTON	cotton	پنبه	pambe	Plants	Plant products	Other plant products
+Anonby-2018-1500-1052	1052			grow (of plants) (intr.)	رشد کردن	rošd kardan	Plants	Plant processes	
```

Comment:
3461 GROW PLANTS would be more specific.

Reply:
I thought GROW might be a tad bit too broad, but it would be a better mapping than GROW PLANTS as the latter is specifically a transitive action done by humans.

Reply:
Ah, yes. Then this should be mapped to GROW.